src/LockToVotePlugin.sol

```diff
@@ -147,6 +147,11 @@ contract LockToVotePlugin is ILockToVote, MajorityVotingBase, LockToGovernBase {
         override
         auth(LOCK_MANAGER_PERMISSION_ID)
     {
+        /// @dev The DAO can disable auth(LOCK_MANAGER_PERMISSION_ID) from above but not define an arbitrary address other than LockManager
+        if (msg.sender != address(lockManager)) {
+            revert VoteCastForbidden(_proposalId, _voter);
```

Comment:
@brickpop I think that the revert error and the parameters are not useful if I need to be honest. 

It seems like the issue (the reason why it has reverted) is relative to the `_proposalId` or the `_voter`, while instead the problem is that the caller (`msg.sender`) is not the `lockManager`.

Reply:
Same comment relative to the same revert error in the `clearVote` function